docs/en/stack/security/get-started-security.asciidoc

```diff
@@ -0,0 +1,311 @@
+communicate with the cluster, you must specify a username and password.
+Unless you <<anonymous-access,enable anonymous access>>, all requests that don't 
+include a user name and password are rejected.
+
+[float]
+[[gs-built-in-users]]
+=== Create passwords for built-in users
+
+There are built-in users that you can use for specific administrative purposes:
+`elastic`, `kibana`, `logstash_system`, and `beats_system`. 
+
+Before you can use them, you must set their passwords. Restart {es}, then run 
+the following command from the {es} directory:
+
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+bin/elasticsearch-setup-passwords interactive
+----------------------------------------------------------------------
+
+You need these built-in users in subsequent steps, so choose passwords that you 
+can remember!
+
+[float]
+[[gs-kibana-security]]
+=== Add the built-in user to {kib}
+
+When {security} is enabled, users must log in to {kib} with a valid user ID and 
+password. 
+
+{kib} also performs some tasks under the covers, which require use of the 
+built-in `kibana` user. Uncomment and update the following settings in the 
+`config/kibana.yml` file in your {kib} directory:
+
+[source,yaml]
+----
+elasticsearch.username: "kibana"
+elasticsearch.password: "your_password"
+----
+
+Specify the password that you set with the `elasticsearch-setup-passwords` 
+command. 
+
+Save your changes to the file and restart {kib}.
+
+[float]
+[[gs-realms]]
+=== Configure authentication
+
+Now that you've set up the built-in users, you need to decide how you want to 
+manage all the other users. 
+
+The {stack} _authenticates_ users to ensure that they are valid. The 
+authentication process is handled by _realms_. You can use one or more built-in 
+realms, such as the native, file, LDAP, PKI, Active Directory, or SAML realms. 
+Alternatively, you can create your own custom realms. In this tutorial, we'll  
+use a native realm. 
+
+In general, you configure realms by adding `xpack.security.authc.realms` 
+settings in the `elasticsearch.yml` file. However, the native realm is available 
+by default when no other realms are configured. Therefore, you don't need to do 
+any extra configuration steps in this tutorial. You can jump straight to 
+creating users!
+
+If you want to learn more about authentication and realms, see 
+<<setting-up-authentication>>.
+
+[float]
+[[gs-users]]
+=== Create users in {kib}
+
+Log in to {kib} with the `elastic` built-in user. 
+
+Go to the *Management / Security / Users* page:
+
+[role="screenshot"]
+image::security/images/management-builtin-users.jpg["User management screenshot in Kibana"]
+
+In this example, you can see all of the built-in users. 
+
+Click *Create user* to make a new user. For example, create a user for yourself:
+
+[role="screenshot"]
+image::security/images/create-user.jpg["Creating a user in Kibana"]
```

Comment:
Is it worth mentioning that the roles are explained in an upcoming step?
I was surprised that there was no explanation here, until I scrolled down a little.

Reply:
You're right, I've revamped so we tell them it's coming and then explicitly add the kibana_user role.